src/html/components/footer.html

```diff
@@ -0,0 +1,9 @@
+<footer>
+  <div class="logo-with-title">
+    <h2>An Exploration By</h2>
+    <img src="images/viget-logo.svg" alt="Viget Logo" class="logo">
+  </div>
+  <p>At Viget, we’re continually thinking about how we can help shape and improve the experiences around us and make the world a better place — whether that’s through the technology solutions we deliver to our clients or through conceptual explorations like Florence.</p>
+  <a href="https://www.viget.com/contact/"><button class="contact">Contact Us</button></a>
```

Comment:
It's not necessary to nest anchor and button here. Just have a single anchor for the link.